misc/python/materialize/output_consistency/enum/enum_operation_param.py

```diff
@@ -0,0 +1,38 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+from typing import Set
+
+from materialize.output_consistency.data_type.data_type import DataType
+from materialize.output_consistency.data_type.data_type_category import DataTypeCategory
+from materialize.output_consistency.enum.enum_constant import EnumConstant
+from materialize.output_consistency.enum.enum_data_type import EnumDataType
+from materialize.output_consistency.operation.operation_param import OperationParam
+
+
+class EnumConstantOperationParam(OperationParam):
+    def __init__(self, values: Set[str], add_quotes: bool, optional: bool = False):
+        super().__init__(
+            DataTypeCategory.ENUM,
+            optional=optional,
+            incompatibilities=None,
+            incompatibility_combinations=None,
+        )
+        self.values = list(values)
+        self.value_count = len(values)
```

Comment:
Why store this? `len(values)` directly should be fast and then it can't be out-of-sync with the `values`